Makefile

```diff
@@ -34,18 +34,16 @@ check_deps:                                                  ## Verify the syste
 	@echo "all dependencies satisifed: $(DEPS)"
 .PHONY: check_deps
 
-build_deps:
-	@command -v goveralls || go get github.com/mattn/goveralls
-	@command -v golint || go get golang.org/x/lint/golint
-	@go tool cover -V || go get golang.org/x/tools/cmd/cover
+build_deps:                                                  ## Install all dependencies specified in tools.go
+	@grep _ tools.go | cut -d '"' -f 2 | xargs go install
```

Comment:
I cannot believe there's no better way to tell go to install all modules specified in go.mod...

Reply:
Doesn't this require network connectivity on every run?

Reply:
No, it uses the module cache. The first may download modules, but subsequent invocations do not.